docs/standard/native-interop/best-practices.md

```diff
@@ -19,7 +21,11 @@ The guidance in this section applies to all interop scenarios.
 - ✔️ DO only use `[In]` and `[Out]` attributes on other types when the behavior you want differs from the default behavior.
 - ✔️ CONSIDER using <xref:System.Buffers.ArrayPool%601?displayProperty=nameWithType> to pool your native array buffers.
 - ✔️ CONSIDER wrapping your P/Invoke declarations in a class with the same name and capitalization as your native library.
-  - This allows your `[DllImport]` attributes to use the C# `nameof` language feature to pass in the name of the native library and ensure that you didn't misspell the name of the native library.
+  - This allows your `[LibraryImport]` or `[DllImport]` attributes to use the C# `nameof` language feature to pass in the name of the native library and ensure that you didn't misspell the name of the native library.
+
+## LibraryImport attribute settings
+
+A Roslyn Analyzer will help guide users with `LibraryImportAttribute`. In most circumstances, `LibraryImportAttribute` uses require explicit declarations rather than relying on default settings. This design is intentional and helps avoid unintended behavior in interop scenarios.
```

Comment:
```suggestion
A code analyzer, with ID [SYSLIB1054](../../fundamentals/syslib-diagnostics/syslib1050-1069.md), helps guide you with `LibraryImportAttribute`. In most cases, the use of `LibraryImportAttribute` requires an explicit declaration rather than relying on default settings. This design is intentional and helps avoid unintended behavior in interop scenarios.
```